### ISSUE TITLE
Left UI Shell: The main content is obscured, both through a failure to reflow and the lightbox effect.

### ISSUE THREAD

**Issue by tay1orjones** [carbon-design-system/carbon#13415]
From the parent issue: 

> Problem 5: The main content is obscured, both through a failure to reflow and the lightbox effect.
> When the hamburger menu is activated, two things happen to exacerbate the issue.
> 
> There is a lightbox effect, which dims all the main content
> The main content is not adjusted so that it reflows into the space not obscured by the left nav. Instead it continues to display across the whole width of the screen, despite being partially non-visible.
> This means that the main content is both made less visible due to contrast and partially fully obscured behind the left navigation. This will be a failure in [WCAG 2.2 (Focus Not Obscured)](https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-minimum.html), and is not a great experience!
> 
> These problems are compounded, and there are several ways to alleviate them, some of which solve more than one problem at once. Carbon needs to decide what exactly the responsive design interaction is supposed to be. Happy to discuss the possibilities -- both optimal and acceptable -- from an accessibility perpspective!
